reports/tasks.py

```diff
@@ -205,37 +204,16 @@ def get_registrations(self, **kwargs):
 
     def get_subscriptions(self, sbm_client, **kwargs):
         subscriptions = sbm_client.get_subscriptions(kwargs)
-        cursor = subscriptions['next']
-        while cursor:
-            for result in subscriptions['results']:
-                yield result
-            params = parse_cursor_params(cursor)
-            subscriptions = sbm_client.get_subscriptions(params)
-            cursor = subscriptions['next']
         for result in subscriptions['results']:
             yield result
```

Comment:
Can we not just do a `return sbm_client.get_subscriptions(kwargs)['results']` here?

Reply:
I removed these methods altogether and just called the client directly